plugins/imfile/imfile.c

```diff
@@ -1384,6 +1452,9 @@ pollFile(act_obj_t *const act)
 {
 	cstr_t *pCStr = NULL;
 	DEFiRet;
+	if (act->is_symlink) {
+		RETiRet;    /* no reason to poll symlink file */
```

Comment:
This should be FINALIZE (RETiRet must only be used at end of function, in some settings it can generate extra code).